### PR TITLE
New Page About Broccoli 

### PR DESCRIPTION
**Pull request by karlwithak**
Adds a new html and css file for the broccoli page

image

### CODE REVIEW

File: code-reviews/example1/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
+        <h2>Broccoli 🥦</h2>
```
Comment (karlwithak) [inline code-reviews/example1/index.html:10]: I think we should remove the emoji!

**Review by karlwithak — COMMENTED** [review]
looks great!

**Comment by Griffins2005** [comment]
looks great

File: code-reviews/example1/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <p>A humble and delicious vegetable</p>
+        <p>Here are my favourite things about it</p>
+        <ol class='list'>
+            <li>Tastes Great</li>
```
Comment (collinsmathinji) [inline code-reviews/example1/index.html:14]: you sure its tasty?

File: code-reviews/example1/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            <li>Nutritious</li>
+            <li>It's green!</li>
+            <li>Delicious</li>
+        </ol>
+        <H3>What you can do with broccoli</H3>
```
Comment (theeurbanlegend) [inline code-reviews/example1/index.html:19]: Will the code still look grammatically nice if you use h3 tag in capital letters?

File: code-reviews/example1/styles.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.main {
+    background-color: #add8e6;
+    color: black;
```
Comment (Mwangangi1978) [inline code-reviews/example1/styles.css:3]: maybe white would be better

File: code-reviews/example1/index.html (hunk 3 above)
Comment (collinsmathinji) [inline code-reviews/example1/index.html:19]: h3 why use capital?

Comment (Leevy-Otieno) [reply code-reviews/example1/index.html:19]: h3 should start in lower case

Comment (jjibrah) [inline code-reviews/example1/index.html:19]: are you sure you have used the correct h3 tag

File: code-reviews/example1/styles.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    padding: 12px;
+}
+
+.list {
+    font-weight: bold;
+    margin-bottom: 4;
+}
+
+recipes {
```
Comment (Pkorirdarius) [inline code-reviews/example1/styles.css:12]: The class recipe has not been referenced well:I think it should be .recipes ,The dot was not included

File: code-reviews/example1/index.html (hunk 2 above)
Comment (AlvinKab) [inline code-reviews/example1/index.html:14]: ok

File: code-reviews/example1/index.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
```
Comment (Obarajames) [inline code-reviews/example1/index.html:9]: reduce the margin from the top

File: code-reviews/example1/index.html (hunk 1 above)
Comment (SimonSallstrom) [inline code-reviews/example1/index.html:10]: Good choice! I Like Broccoli :)

File: code-reviews/example1/index.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
```
Comment (theeurbanlegend) [inline code-reviews/example1/index.html:5]: what happens if the user requires to view it on a smaller screen device/could u add a metatag for viewport width

Comment (collinsmathinji) [inline code-reviews/example1/index.html:5]: why not put an icon for your webpage?

File: code-reviews/example1/index.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (Willylaisa) [inline code-reviews/example1/index.html:4]: I think there are some missing meta tags...
For instance, I'm not really sure about this but I think web developers add their names and the following:
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

File: code-reviews/example1/index.html (hunk 2 above)
Comment (AlvinKab) [inline code-reviews/example1/index.html:14]: ok

File: code-reviews/example1/index.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+            <li>Tastes Great</li>
+            <li>Nutritious</li>
+            <li>It's green!</li>
+            <li>Delicious</li>
+        </ol>
+        <H3>What you can do with broccoli</H3>
+        <p class='recipes'>
+            There's so much you can do with broccoli!
+            I love roasting it, and eating it in salads.
+            Please email me with your favourite recipes!
```
Comment (Obarajames) [inline code-reviews/example1/index.html:23]: Add  a link to enable people to be able send email

File: code-reviews/example1/index.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
+        <h2>Broccoli 🥦</h2>
+        <p>A humble and delicious vegetable</p>
```
Comment (Griffins2005) [inline code-reviews/example1/index.html:11]: looks great.

File: code-reviews/example1/styles.css
Comment (G3orge0tevin) [inline code-reviews/example1/styles.css:1]: 👍

File: code-reviews/example1/styles.css [hunk 11]
```diff
@@ -0,0 +1,15 @@
+.main {
+    background-color: #add8e6;
```
Comment (Brikita) [inline code-reviews/example1/styles.css:2]: I dont like the color

File: code-reviews/example1/index.html (hunk 1 above)
Comment (Willylaisa) [inline code-reviews/example1/index.html:10]: Why didn't you start with a h1?

File: code-reviews/example1/index.html (hunk 3 above)
Comment (Willylaisa) [inline code-reviews/example1/index.html:19]: Why did you use capital in the tags? I think it's syntactically wrong... I'm not sure though...

**Review by G3orge0tevin — COMMENTED** [review]
👍

File: code-reviews/example1/index.html (hunk 2 above)
Comment (AlvinKab) [inline code-reviews/example1/index.html:14]: ok

**Review by jjibrah — CHANGES_REQUESTED** [review]
Your should check on your h3 tag

File: code-reviews/example1/index.html (hunk 2 above)
Comment (SimonSallstrom) [inline code-reviews/example1/index.html:14]: What do you think about doing 3 things instead? As they say, all good things come in 3 :)

File: code-reviews/example1/styles.css (hunk 5 above)
Comment (theeurbanlegend) [inline code-reviews/example1/styles.css:12]: Will the styles really apply when the class "styles" is referenced without a dot at the start?

File: code-reviews/example1/index.html (hunk 3 above)
Comment (t-ed-c) [reply code-reviews/example1/index.html:19]: sure thing

File: code-reviews/example1/index.html [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+        <p>Here are my favourite things about it</p>
```
Comment (RealTim0) [inline code-reviews/example1/index.html:12]: what about including a colon at the end?

File: code-reviews/example1/styles.css (hunk 4 above)
Comment (Mwangangi1978) [inline code-reviews/example1/styles.css:3]: mabe white would be better?

File: code-reviews/example1/index.html (hunk 7 above)
Comment (itsbluejelly) [inline code-reviews/example1/index.html:5]: what if you add the meta viewport tag? I think it can help with screen responsiveness

**Review by patrickleon710 — COMMENTED** [review]
Could you change the color?

File: code-reviews/example1/index.html (hunk 3 above)
Comment (RealTim0) [inline code-reviews/example1/index.html:19]: Shouldn't the H in H3 tag be lower case?

File: code-reviews/example1/index.html (hunk 6 above)
Comment (itsbluejelly) [inline code-reviews/example1/index.html:9]: Why did u enclose the content in a div tag? I don't think it affects the code in any way

**Review by Willylaisa — COMMENTED** [review]
Great work though!!!

File: code-reviews/example1/styles.css (hunk 5 above)
Comment (RealTim0) [inline code-reviews/example1/styles.css:12]: what about putting it as .recipes as it is a a class?

**Review by Willylaisa — COMMENTED** [review]
Great work though!!!

**Review by Brikita — COMMENTED** [review]
Its okay

**Review by Cal-Davis — COMMENTED** [review]
How about you use a list to display the things you can do with broccoli (under class ="recipes")

File: code-reviews/example1/index.html [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+        <ol class='list'>
+            <li>Tastes Great</li>
+            <li>Nutritious</li>
+            <li>It's green!</li>
+            <li>Delicious</li>
+        </ol>
+        <H3>What you can do with broccoli</H3>
+        <p class='recipes'>
+            There's so much you can do with broccoli!
+            I love roasting it, and eating it in salads.
```
Comment (lizziegitau) [inline code-reviews/example1/index.html:22]: I didn't know you can roast broccoli

File: code-reviews/example1/index.html (hunk 1 above)
Comment (t-ed-c) [inline code-reviews/example1/index.html:10]: thats a tree emoji

File: code-reviews/example1/index.html (hunk 3 above)
Comment (Jhn-Gi) [reply code-reviews/example1/index.html:19]: True

File: code-reviews/example1/index.html [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+            Please email me with your favourite recipes!
+        </p>
+    </div>
+</body>
+</html>
```
Comment (t-ed-c) [inline code-reviews/example1/index.html:27]: looking good

**Comment by karlwithak** [comment]
Closing, so I can re-open for new cohort!

**Pull request closed without merging**